scss/forms/_input-group.scss

```diff
@@ -107,6 +107,7 @@ $input-prefix-padding: 1rem !default;
 
     @if $global-flexbox {
       flex: 0 0 auto;
+      display: flex;
```

Comment:
Display first and then `flex` shorthand property 
See https://github.com/zurb/foundation-sites/blob/develop/.sass-lint.yml#L151-L153